torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -622,6 +623,34 @@ Tensor _sparse_addmm_sparse_backward(const Tensor& grad, const Tensor& sparse_,
   return grad_sparse.sparse_mask(sparse);
 }
 
+Tensor _sparse_matrix_mask(const Tensor& grad, const Tensor& mask){
+  Tensor output = at::native::empty_like(mask);
+  if (mask._nnz() == 0) {
+    return output.zero_();
+  }
+  Tensor mask_indices = at::empty_like(mask._indices());
```

Comment:
It seems like you can just clone mask._indices() here instead of using empty_like and copy_